content/en/software/install-and-run/windows/native/_index.md

```diff
@@ -71,24 +76,23 @@ This will bring up the Medley environment in a separate Window on your Windows d
 
 ![Medley window open on Windows desktop](../images/Medley-on-Windows.png)
 
-
-
-
 ## Use Medley
+
 Once Medley is up and running, see [here](/software/using-medley/) for tips on how to navigate and use the Medley environment.
 
-To exit Medley, type ```(IL:LOGOUT)``` at any Exec window prompt.
+To exit Medley, type `(IL:LOGOUT)` at any Exec window prompt.
 
 ## Important Notes
-1. The file system from Medley's perspective differs somewhat from the file system from Window's perspective.   Specifically, within Medley, the file system root (i.e., "{DSK}/" or "{UNIX}/") is mapped to the ***install_dir*** in the Windows file system.  For example, if ***install_dir*** is ***C:\Users\Frank\Medley1***, then the Medley file known as ***{DSK}\<testdir>testfile.txt*** will be located in the Windows file system at  ***C:\Users\Frank\Medley1\testdir\testfile.txt*** 
+
+1. The file system from Medley's perspective differs somewhat from the file system from Window's perspective.   Specifically, within Medley, the file system root (i.e., "{DSK}/" or "{UNIX}/") is mapped to the ***install_dir*** in the Windows file system.  For example, if ***install_dir*** is ***C:\Users\Frank\Medley1***, then the Medley file known as ***{DSK}\<testdir>testfile.txt*** will be located in the Windows file system at  ***C:\Users\Frank\Medley1\testdir\testfile.txt***
 
 2. Also, from Medley you can refer to any file on a Windows drive using the reference ***{DSK}/[drive letter]/*** (or equivalently ***{DSK}<[drive letter]>***).  For example, ***C:\User\Frank\Downloads\testfile.txt*** in Windows will be  ***{DSK}/c/User/Frank/Downloads/testfile.txt*** in Medley.
 
```

Comment:
unfortunately this section depends on "native" vs "wsl". With wsl, the drive letter is prefixed with `/mnt/<driveletter>`, e.g.,   
```
   {DSK}/mnt/d/Users/Frank/Downloads/testfile.txt
or
   {DSK}<mnt>c>Users>Frank>Downloads/testfile.txt
```

```

Reply:
OK - my changes in this section were only editorial, cleaning up some of the markdown and using it to validate the code highlighting was working correctly - the addition of `powershell` on lines 53 and 63 will cause the markdown the code blocks to have syntactical highlighting for powershell.  The blank lines I added are bring the formatting into compliance with standard markdown formatting.